src/sql/finalize.sql

```diff
@@ -625,6 +681,22 @@ CREATE AGGREGATE sum(vector) (
     PARALLEL = SAFE
 );
 
+CREATE AGGREGATE avg(svector) (
+	SFUNC = _vectors_svector_accum,
+	STYPE = svector_accumulate_state,
+	COMBINEFUNC = _vectors_svector_combine,
+	FINALFUNC = _vectors_svector_final,
+	INITCOND = '(0, [0])',
```

Comment:
I tried to use `(0, NULL`). But I don't find a way to use `NULL` for `INITCOND`;

Reply:
`(0,)`

Reply:
get! thanks.